wiki/Help_centre/Upgrading_to_lazer/it.md

```diff
@@ -0,0 +1,284 @@
+
+#### I punteggi impostati su stable saranno visualizzati in lazer?
+
+Sì. Una volta terminato il bilanciamento della combinazione di punteggi in lazer e in stable, entrambi saranno visibili.
+
+#### Tutte le mod saranno Classificate?
+
+Per ora, i punteggi di tutte le combinazioni di mod appaiono nelle classifiche. Si sta ancora discutendo se i punteggi daranno punti performance con tutte le mod (e se lo faranno, se ci sarà un bonus o una penalità).
+
+#### Non mi piacciono le nuove meccaniche di gioco. Posso ripristinare le vecchie meccaniche di gioco come su stable?
+
+Prova ad applicare la mod "classic", che ripristinerà gran parte delle vecchie meccaniche di gioco a cui sei abituato. Assicurati anche di controllare le impostazioni offerte dalla mod classica, in quanto ti consentiranno di personalizzare ulteriormente la tua esperienza e di capire quali modifiche vengono applicate (in quanto sono tutte elencate).
+
+### Skinning e UI
+
+#### Qualcosa si comporta in modo diverso rispetto a stable e non mi piace!
+
+Eseguire la configurazione guidata nella parte superiore delle impostazioni e controllare le impostazioni nella schermata `Comportamento`. Qui sono elencate molte delle impostazioni comuni che sono state modificate per impostazione predefinita. C'è anche un singolo pulsante che si può premere per applicare i vecchi comportamenti come punto di partenza per il vostro viaggio con il lazer.
+
+#### le vecchie skin funzioneranno nella selezione delle mappe e nella schermata del risultato?
+
+Faremo del nostro meglio per riportare queste funzionalitá 
+
+#### posso usare il cursore della skin pure sul menù?
+
+porteremo molto probabilmente questa novità date la richiesta elevata.
+
+### Prestazioni
+
+#### perchè non posso avere gli fps illimitati?
+
+Al di sopra di una certa soglia, non c'è motivo di utilizzare frame rate più elevati. Lazer impiega diverse nuove tecnologie per garantire la latenza più bassa possibile senza richiedere frame rate elevati. Questo aspetto continuerà a migliorare in futuro, poiché abbiamo ancora ulteriori miglioramenti da implementare.
+
+Lazer esegue il polling rate in ingresso a 1000 Hz indipendentemente dal limitatore di FPS, motivo per cui anche l'impostazione massima del limitatore limiterà a 1000 FPS.
+
+Se siete curiosi di sapere come questo influisce sulla latenza in ingresso e di verificare la vostra percezione, eseguite il "certificatore di latenza" integrato in fondo alle impostazioni.
+
+#### Se il polling dell'input viene eseguito solo a 1000 Hz, che ne è del mio mouse da gioco a 8000 Hz?
+
+Il sistema operativo continuerà a eseguire il polling alla frequenza più alta possibile, anche se i vantaggi sono dimostrati essere irrilevanti. Il polling a velocità così elevate può comportare richiesta di potenza aggiuntiva imprevista e si consiglia di limitare le proprie periferiche a 1000 Hz per garantire una esperienza stabile.
+
+#### Lazer non funziona in modo stabile. Cosa succede?
```

Comment:
Doesn't look like the original.
I suggest: "Lazer performa peggio di stable per me. Che succede?"